deps/build.jl

```diff
@@ -51,7 +51,7 @@ provides(BuildProcess,
 )
 
 
-@osx_only begin
+@compat @static if is_apple()
```

Comment:
I'm being picky here, but the explicit `@compat` shouldn't be required here - these are not syntax incompatibilities, but API incompatibilities.  So `Compat` will define both `unsafe_string` and `@static` for older versions.